openlibrary/templates/lib/nav_foot.html

```diff
@@ -47,6 +47,17 @@ <h2>$:_('Help')</h2>
           <a class="footer-icons__github" href="https://github.com/internetarchive/openlibrary">github</a>
         </aside>
       </div>
+      <div id="footer-locale-menu">
+        <h2>$:_('Change Website Language')</h2> 
+        <ul id="locale-options">
+          <li id="cs"><a href="/" title="Čeština">Čeština (cs)</a> </li>
+          <li id="de"><a href="/" title="Deutsch">Deutsch (de)</a> </li>
+          <li id="en"><a href="/" title="English">English (en)</a></li>
+          <li id="es"><a href="/" title="Español">Español (es)</a></li>
+          <li id="fr"><a href="/" title="Français">Français (fr)</a></li>
+          <li id="te"><a href="/" title="తెలుగు">తెలుగు (te)</a></li>
```

Comment:
These `id` are too short; they're likely to conflict. We can use a data param:

```suggestion
          <li><a href="#" data-lang-id="cs">Čeština (cs)</a></li>
          <li><a href="#" data-lang-id="de">Deutsch (de)</a></li>
          <li><a href="#" data-lang-id="en">English (en)</a></li>
          <li><a href="#" data-lang-id="es">Español (es)</a></li>
          <li><a href="#" data-lang-id="fr">Français (fr)</a></li>
          <li><a href="#" data-lang-id="te">తెలుగు (te)</a></li>
```

Also: It's best practice to not include a `title` if it's the same as the text content, since screenreaders might read it twice.

Also instead of going to the homepage, we likely want these to remain on the current page. So I set the `href` to `#`, and moved the `data-lang-id` to the `<a>` element so we can bind a click event handler to that, and then use `event.preventDefault()` and just trigger a `window.reload()` instead.

Reply:
Done